client/ayon_maya/plugins/publish/extract_layout.py

```diff
@@ -176,6 +178,34 @@ def process(self, instance):
         self.log.debug("Extracted instance '%s' to: %s",
                        instance.name, json_representation)
 
+    def get_containers_sub_assembly(self, containers, asset):
+        """Allow to collect the asset containers through sub-assembly workflow if
+        there is a layout container
+
+        Args:
+            containers (list): Ayon asset containers
+            asset (str): asset name
+
+        Returns:
+            dict: container dict with the container and its related asset name
+        """
+        all_containers_set = {}
+        for existing_container in containers:
+            if cmds.getAttr(f"{existing_container}.loader").lower() == "layoutloader":
```

Comment:
```suggestion
            # Flatten layout loader to their individual loaded containers
            if cmds.getAttr(f"{existing_container}.loader").lower() == "layoutloader":
```

Also, why are we doing a `lower()`comparison? Don't we know the _exact_ plug-ins name?

```suggestion
            # Flatten container layout loader products to their individual loaded containers
            if cmds.getAttr(f"{existing_container}.loader") == "LayoutLoader":
```